### DISCUSSION TITLE
Live Rotation

### DISCUSSION THREAD

**Discussion by luisesolisgomez** [bluenviron/mediamtx#3396] [Questions]
### Question

Hey, I was wondering if there is way to rotate the stream via key inputs and be able to stream at the same time? Perhaps in C++?

**Comment by aler9** [comment]
Hello, yes, you can use OBS studio with the Source Switcher feature:
https://www.youtube.com/watch?v=fF7Sq6hEeEw

[a comment by an automated account is not shown]
